core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceTest.java

```diff
@@ -793,7 +783,7 @@ public void testParam() throws Exception {
 
     @Test
     public void testAdvancedAnnotatedHttpService() throws Exception {
```

Comment:
Could you remove all `public` modifiers in this class as well?

Reply:
I was remove `public`